### DISCUSSION TITLE
Add text to images with ImageSharp.Web

### DISCUSSION THREAD

**Discussion by olaj** [SixLabors/ImageSharp.Web#131] [Q&A] [answered]
Is it possible? Will it be possible anytime soon? :) Like with a simple querystring api?

**Comment by JimBobSquarePants** [comment, marked as answer]
Only if you implemented your own `IImageWebProcessor` and referenced ImageSharp.Drawing. 

The default API is limited by design to reduce the potential out of the box issues with DDOS attacks.
